app/generators/date_and_time_generator.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+class DateAndTimeGenerator < QuestionTypeGenerator
+  include React::Rails::ViewHelper
+  DATETIMEFIELD_PLACEHOLDER = { nl: 'Vul een datum en tijd in', en: 'Fill out a date and time' }.freeze
+
+  def generate(question)
+    title = safe_join([question[:title].html_safe, generate_tooltip(question[:tooltip])])
+    safe_join([tag.p(title, class: 'flow-text'), mydate_and_time_field(question)])
+  end
+
+  private
+
+  def mydate_and_time_field(question)
+    body = mydate_and_time_tag(question)
+    body = tag.div(body, class: 'col s12 m6')
+    tag.div(body, class: 'row')
+  end
+
+  # rubocop:disable Metrics/AbcSize
+  def mydate_and_time_tag(question)
+    locale = (question[:locale].presence || Rails.application.config.i18n.default_locale).to_sym
```

Comment:
just double checking, that `.presence` is correct here?

Reply:
Yep. It `question[:locale].presence` is equivalent to `question[:locale] if question[:locale].present?`.

It was actually rubocop that automatically changed it into this for me.